src/helpers/filters.ts

```diff
@@ -53,15 +53,20 @@ const filterByFromDate = (transactions: Transaction[], fromDate: number) => {
 
 const filterByToDate = (transactions: Transaction[], toDate: number) => {
   return transactions.filter(transaction => {
-    if (!transaction.time) {
+    if (!transaction.received) {
       return;
     }
     return (
       parseInt(
         moment(toDate)
           .endOf('day')
           .format('X'),
-      ) >= transaction.time
+      ) >=
```

Comment:
check the function https://momentjscom.readthedocs.io/en/latest/moment/05-query/01-is-before/